timeline-chart/src/layer/time-graph-layer.ts

```diff
@@ -73,7 +73,8 @@ export abstract class TimeGraphLayer {
     protected afterAddToContainer() { }
 
     destroy() {
-        this.removeChildren();
+        this.layer.destroy();
```

Comment:
removeChildren() calls destroy() on each child component which destroys the corresponding display object and WebGL resource. Not calling destroy() would cause WebGL resources not being disposed.

Reply:
Now calling destroy({children: true}) to destroy all WebGL resources of the layer and its children components recursively.

If there are any user-application resources to release in any layer it can be done in the layer's destroy() method.

With the changes, there are no more user-application resources to release in any of the components. If it's needed in the future I do not think it should be done through the component's destroy() method.